readme.md

```diff
@@ -85,495 +71,355 @@ examples to which eBPF brings performance, programmability and flexibility.
-    -   [XDP for the Rest of Us](http://netdevconf.org/2.1/session.html?gospodarek)
-        (netdev 2.1, Montreal, April 2017), with Andy Gospodarek.
-        How to get started with eBPF and XDP for normal humans.
+    - [Memory vs. Networking, Provoking and fixing memory bottlenecks](http://people.netfilter.org/hawk/presentations/MM-summit2017/MM-summit2017-JesperBrouer.pdf) - Advanced details about current memory issues faced by XDP developers.
+    - [XDP for the Rest of Us](http://netdevconf.org/2.1/session.html?gospodarek) - How to get started with eBPF and XDP for normal humans.
         Also summarized by Julia Evans on
         [her blog](http://jvns.ca/blog/2017/04/07/xdp-bpf-tutorial/).
-    -   [XDP now with REDIRECT](http://people.netfilter.org/hawk/presentations/LLC2018/XDP_LLC2018_redirect.pdf)
-        (LLC, Lund (Sweden), May 2018)
-        Update on XDP, and in particular on the redirect actions.
--   [XDP workshop — Introduction, experience, and future development](http://netdevconf.org/1.2/session.html?herbert-xdp-workshop)
-    (Tom Herbert, netdev 1.2, Tokyo, October 2016) (Video).
--   [High Speed Packet Filtering on Linux](https://cdn.shopify.com/s/files/1/0177/9886/files/phv2017-gbertin.pdf)
-    (Gilberto Bertin, DEF CON 25, Las Vegas, July 2017) About packet filtering
+    - [XDP now with REDIRECT](http://people.netfilter.org/hawk/presentations/LLC2018/XDP_LLC2018_redirect.pdf) - Update on XDP, and in particular on the redirect actions.
+- [XDP workshop — Introduction, experience, and future development (Video)](http://netdevconf.org/1.2/session.html?herbert-xdp-workshop)
+- [High Speed Packet Filtering on Linux](https://cdn.shopify.com/s/files/1/0177/9886/files/phv2017-gbertin.pdf) - About packet filtering
     on Linux, DDoS protection, packet processing in the kernel, kernel bypass,
     XDP and eBPF.
 
 ### AF\_XDP
 
--   [AF\_XDP](https://www.kernel.org/doc/html/latest/networking/af_xdp.html)
-    Kernel documentation on the AF\_XDP address family.
--   [Fast Packet Processing in Linux with AF\_XDP](https://archive.fosdem.org/2018/schedule/event/af_xdp/)
-    (Björn Töpel and Magnus Karlsson, FOSDEM 2018, Brussels, February 2018).
+- [AF\_XDP](https://www.kernel.org/doc/html/latest/networking/af_xdp.html) - Kernel documentation on the AF\_XDP address family.
+- [Fast Packet Processing in Linux with AF\_XDP](https://archive.fosdem.org/2018/schedule/event/af_xdp/)
 
 ### bpfilter
 
--   [A high-level post](https://cilium.io/blog/2018/04/17/why-is-the-kernel-community-replacing-iptables/)
-    by Thomas Graf about the motivations behind this project.
--   [A brief introduction](https://qmo.fr/docs/talk_20180316_frnog_bpfilter.pdf)
-    by Quentin Monnet at FRnOG 30.
+- [A high-level post](https://cilium.io/blog/2018/04/17/why-is-the-kernel-community-replacing-iptables/) - About the motivations behind this project.
+- [A brief introduction](https://qmo.fr/docs/talk_20180316_frnog_bpfilter.pdf)
```

Comment:
For the two links above we probably want to change the link title. Use the title of the presentations, or at least something more complete like “a brief introduction to bpfilter”? Just “a brief introduction” sounds kind of weird.